source/release-notes.txt

```diff
@@ -10,6 +10,27 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.2.0 Changelog
+---------------
+
+*Released September 4, 2023*
+
+New features:
+
+- Ability to change the data type for migrated fields.
+- Ability to control how null values are represented in the migrated data.
+- Ability to use calculated fields for a collection's ``_id`` value.
+- Diagnostic dump includes configuration files to help streamline support. 
+
+Bug fixes:
+
+- Improvements to validation and error handling in mapping rules.
```

Comment:
[suggestion] For parallelism, maybe we can start the bullets here to start with verbs and change this bullet to:
```suggestion
- Improves validation and error handling in mapping rules.
```

Reply:
Similar suggestion for the bullet below!

Reply:
Done, thank you!